amp/TEMPLATE/reampv2/src/modules/sscdashboard/utils/exportUtils.js

```diff
@@ -0,0 +1,204 @@
+import Excel from 'exceljs';
+
+/**
+ *
+ * Code from VIFAA project.
+ */
+export const dataDownloadStyles = {
+    totalLabels: {
+        alignment: {
+            vertical: 'middle',
+            horizontal: 'middle'
+        },
+        font: {
+            name: 'Calibri',
```

Comment:
We could move some of the strings to constants if they are going to be used elsewhere.